src/Backend/EdgeTPU/EdgeTPUToolchain.ts

```diff
@@ -77,4 +83,191 @@ class EdgeTPUDebianToolchain extends DebianToolchain {
   }
 }
 
-export { EdgeTPUDebianToolchain };
+class EdgeTPUCompiler implements Compiler {
+  private readonly toolchainTypes: string[];
+  private readonly toolchainName: string;
+
+  constructor() {
+    this.toolchainName = "edgetpu-compiler";
+    this.toolchainTypes = ["latest"];
+  }
+
+  getToolchainTypes(): string[] {
+    return this.toolchainTypes;
+  }
+
+  parseVersion(version: string): Version {
+    if (!version.trim()) {
+      throw Error("Invalid version format.");
+    }
+
+    let _version = version;
+    let option = "";
+
+    const optionIndex = version.search(/[~+-]/);
+    if (optionIndex !== -1) {
+      option = version.slice(optionIndex);
+      _version = version.slice(0, optionIndex);
+    }
+
+    const splitedVersion = _version.split(".");
+
+    if (splitedVersion.length > 3) {
+      throw Error("Invalid version format.");
+    }
+
+    let major: number | string;
+    let minor: number | string;
+    let patch: number | string;
+
+    [major = "0", minor = "0", patch = "0"] = _version.split(".");
+
+    const epochIndex = major.search(/:/);
+    if (epochIndex !== -1) {
+      major = major.slice(epochIndex + 1);
+    }
+
+    major = Number(major);
+    minor = Number(minor);
+    patch = Number(patch);
+
+    if (isNaN(major) || isNaN(minor) || isNaN(patch)) {
+      throw Error("Invalid version format.");
+    }
+
+    if (splitedVersion.length === 2 && !option) {
+      return new Version(major, minor, undefined);
+    }
```

Comment:
Q. What are these lines for? It seems that patch will be '0' by default when the split length is 2.

Reply:
The latest version of EdgeTPU Compiler is 16.0.

Setting the patch version as the default(16.0.0) causes the installation of edgetpu-compiler using aptitude to fail.

Therefore, I have implemented it to be set as undefined if there is no patch.

However, it seems that if we use undefined in cases where there is no patch version, there may be no need for default value settings.

I will try to modifying it while extracting the common parts of the OneCompiler and EdgeTPUCompiler classes.